functions/google_maps_params.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-console */
+exports.handler = function(event, context, callback) {
+  // Setup development environment (production or sandbox)
+  // In the future, we would like this to be set automatically via environment variables set using deploy context
+  const env = process.env.MAPS_API; // Set in Netlify UI
```

Comment:
We need to make a few changes here. Let's change this line to `const env = process.env.SITE_ENV;`

`SITE_ENV` is another variable stored in Netlify that is set to either "sandbox" or "production". That helps us later on to decide which key to use.

Reply:
Done.